core/domain/stats_jobs_one_off.py

```diff
@@ -329,10 +342,11 @@ def reduce(exp_id, stringified_values):
                 except KeyError:
                     yield (
                         'ERROR: State not in stats mapping exp_id %s, version '
-                        '%s, State %s, state_stats_mapping [%s]' % (
-                            exp_id, version, state_name.encode('utf-8'),
+                        '%s, State %s, Encoded State %s, '
```

Comment:
Were you planning to remove the try/except here? If so I think that's fine.

Reply:
@seanlip Let's actually keep this around. I'm not sure if the encoding state name fixes this - so, it's better to log this if its not actually fixed.

Reply:
OK -- then, I have the same question as below about the removal of the `raise`. Why do we do that and what happens if it's removed and this exception triggers?

Reply:
@seanlip My bad. I had to add a return statement at the end. This exception is not supposed to be raised. This is an issue we need to fix, and so, we return if the exception is raised.